src/nanoemoji/config.py

```diff
@@ -168,18 +169,39 @@ def output_format(self):
         return Path(self.output_file).suffix
 
     @property
-    def has_bitmaps(self):
-        return self.color_format.startswith("sbix") or self.color_format.startswith(
-            "cbdt"
+    def has_bitmaps(self) -> bool:
+        return bool({"sbix", "cbdt"}.intersection(self.color_format.split("_")))
```

Comment:
Consider {sbix, cbdt} & self.color_format.split, to my understanding use of those operators is idiomatic

Reply:
to use the set operators both operands need to be sets, wheras the named methods allow the second operand to be any iterable, in this case a list. That's why I went with those. The alternative would be `bool({"sbix", "cbdt"} & set(self.color_format.split("_")))` which makes a temporary set object for the sake of the intersection and throws that away. It's not more concise either.